test/test-list.h

```diff
@@ -482,14 +483,22 @@ TASK_LIST_START
 
   TEST_ENTRY  (tcp_alloc_cb_fail)
 
-  TEST_ENTRY  (tcp_ping_pong)
+#if defined(__linux__)
+#define PING_PONG_TIMEOUT 30000
+#elif defined(_WIN32)
+#define PING_PONG_TIMEOUT 120000
+#else
+#define PING_PONG_TIMEOUT 60000
+#endif
```

Comment:
Those timeouts are way too high, that makes the test suite unacceptably slow.  Can you drop these changes and add a smaller test that tests a big TCP and/or pipe write?

Reply:
That would still take the same amount of time, if it's having the kernel still do the same amount of work. On a linux system with a good processor, this test should be pretty quick (my macOS laptop finishes it in a couple seconds), but it is heavily dependent on the kernel performance (and clock speed).